typescript.js

```diff
@@ -15,9 +15,21 @@ module.exports = {
         'plugin:@typescript-eslint/eslint-recommended',
         // turns on rules from their TypeScript-specific plugin
         'plugin:@typescript-eslint/recommended',
+        // enables ts/tsx file usage when importing modules
+        'plugin:import/typescript',
     ],
     rules: {
         // Allow TypeScript files to have JSX
         'react/jsx-filename-extension': [WARNING, { 'extensions': ['.ts', '.tsx', '.js', '.jsx'] }],
+        'import/extensions': [
```

Comment:
[question]: this seems like it should be a general rule in some fashion - with the exception of the `ts|x` extensions, this rule could reside in `index.js`. I wonder if we can split this up...or duplicate is fine in this case. What do you think?

Reply:
I'd imagine that we'd enforce this rule generally

Reply:
Hey @InsaneViku, the original `import/extensions` is currently being set by AirBnB's ESLint rules. I couldn't find an existing rule for this in our config, so I placed it here since it's related to TypeScript. If you'd prefer that it be in `index.js`, I can move it over.

Reply:
Big duh on my part! It's all good then, no need to move 🙌

Reply:
Haha, no worries. I had to double-check to make sure that it was AirBnB that was the culprit. 👍